### ISSUE TITLE
site generation seems to fail

### ISSUE THREAD

**Issue by clickot** [ungoogled-software/ungoogled-chromium-binaries#494]
it seems that after the last merges of portableliunx and windows binaries builds the site was not regenerated. 
Also the site generation check with latest two pull requests failed after some time. 
It looks like the site generation job doesn't have enough resources for the or is otherwise disrupted.

**Comment by PF4Public** [comment]
The next build seems to have succeeded, did it not?
![image](https://github.com/ungoogled-software/ungoogled-chromium-binaries/assets/10319700/318ad415-55f3-4b2b-ab08-d94bea5973ff)

**Comment by claudiodekker** [comment]
Seeing the same here: https://github.com/ungoogled-software/ungoogled-chromium-binaries/pull/492/checks?check_run_id=20950014899

According to that, there's not enough compute credits?:
```
⚠️ Not enough compute credits to prioritize tasks!
❗ Failed to start an instance! Failed to pull null image! Repository does not exist or may require authentication.
```

**Comment by clickot** [comment]
the checks for the PRs for 121.0.6167.139 are all failed, seemingly because of the same reason @claudiodekker stated.

**Comment by claudiodekker** [comment]
Yeah, I suppose the Cirrus run is paid/limited on credits each month or something, and that the easy/free way to solve this issue would be to migrate it to Github Actions, but since doing so takes some time/effort, I figured I'd just go ahead and quickly whip something up, and so that's what I did in #498 😆

**Comment by networkException** [comment]
ill see if I find time to work on this
